app.json

```diff
@@ -2,180 +2,150 @@
-      "value": "never",
-      "required": false 
-    },
-    "THEME": {
-      "description": "Browser default theme setting. Valid settings are default, colors, or colorblindfriendly",
-      "value": "default",
-      "required": false 
-    },
-    "ALARM_URGENT_HIGH": {
-      "description": "Browser default urgent high alarm enabled. Valid settings are on or off",
-      "value": "on",
-      "required": false 
-    },
-    "ALARM_HIGH": {
-      "description": "Browser default high alarm enabled. Valid settings are on or off",
-      "value": "on",
-      "required": false
-    },
-    "ALARM_LOW": {
-      "description": "Browser default low alarm enabled. Valid settings are on or off",
-      "value": "on",
+    "DISPLAY_UNITS": {
+      "description": "Preferred BG units for the site:'mg/dl' or 'mmol'. (Note that it is *not* 'mmol/L')",
+      "value": "mg/dl",
       "required": false
     },
-    "ALARM_URGENT_LOW": {
-      "description": "Browser default urgent low alarm enabled. Valid settings are on or off",
-      "value": "on",
+    "ENABLE": {
+      "description": "Plugins to enable for your site. Must be a space-delimited, lower-case list. Include the word 'bridge' here if you are receiving data from the Dexcom Share service. Include 'mmconnect' if you are bridging from the MiniMed CareLink service.",
+      "value": "careportal basal",
       "required": false
     },
-    "ALARM_TIMEAGO_WARN": {
-      "description": "Browser default warn after time of last data exceeds ALARM_TIMEAGO_WARN_MINS alarm enabled. Valid settings are on or off",
-      "value": "on",
+    "MMCONNECT_USER_NAME": {
+      "description": "Your CareLink account username, to receive CGM data from the CareLink service. Also make sure to include 'mmconnect' in your ENABLE line.",
+      "value": "",
       "required": false
     },
-    "ALARM_TIMEAGO_WARN_MINS": {
-      "description": "Browser default minutes since the last reading to trigger a warning.",
-      "value": "15",
+    "MMCONNECT_PASSWORD": {
+      "description": "Your CareLink account password, to receive CGM data from the CareLink service. Also make sure to include 'mmconnect' in your ENABLE line.",
+      "value": "",
       "required": false
     },
-    "ALARM_TIMEAGO_URGENT": {
-      "description": "Browser default urgent warning after time of last data exceeds ALARM_TIMEAGO_URGENT_MINS alarm enabled. Valid settings are on or off",
-      "value": "on",
+    "MMCONNECT_SERVER": {
+      "description": "If you are bridging from the CareLink service, and are anywhere *outside* the US, change this to EU. ('US' or 'EU')",
+      "value": "US",
       "required": false
     },
-    "ALARM_TIMEAGO_URGENT_MINS": {
-      "description": "Browser default minutes since last reading to trigger an urgent alarm.",
-      "value": "30",
-      "required": false
+    "MONGO_COLLECTION": {
+      "description": "The mLab collection where CGM data is stored.",
```

Comment:
mLab is a MongoDB saas. Nightscout should be vendor neutral for MongoDB, so use MongoDB collection instead of mlab.